### ISSUE TITLE
sound not played for first action

### ISSUE THREAD

**Issue by mrfy-angband** [angband/angband#5962]
Using 4.2.5-185-g02b5e5f72 on macosx, I'm noticing that the first action taken doesn't make a sound but the second does.

For example,
Picking up multiple arrows/shots/bolts, sometimes the first item picked up is silent but 2nd and following ones make the quiver pick up sound.

Or switch to tunneling, and the first wall removed doesn't make a sound but following ones do.

**Comment by backwardsEric** [comment]
Which version of macOS are you using when you see this?  I have not been able to reproduce it on macOS 12.7.4.

**Comment by mrfy-angband** [comment]
Using Sonoma 14.4.1  the most recent version. 

I noticed it when picking up ammo left on the floor. If I drop an iron shot, move, drop another, go to a wall, try to dig, the first dig attempt wouldn't make a sound. And if I then go back to my dropped ammo, picking up the first one didn't make a sound either. 

It started failing after updating to the 185 build.  I had Angband-4.2.5-177-g515cbb3e8-osx.dmg still in my downloads directory so I went back to that version this bug did not occur.

But after reinstalling build 185, it's seems to be working now all the time. So we can close this issue for now I think.
